src/assets/stylesheets/components/_auth-logo.scss

```diff
@@ -0,0 +1,17 @@
+.auth-logo {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: rem(320px);
+  height: rem(80px);
```

Comment:
@gutakk Why do we need to set the height here? seems like the content is exceeded the container height 🤔 
![Screen Shot 2564-06-18 at 19 14 24](https://user-images.githubusercontent.com/1772999/122559387-72bc0400-d069-11eb-8d8d-ebee30de56dc.png)

Reply:
Good catch. Fixed in 6581b37ffa0ae2facd302f4c4eb23620aa73f974
Actually, I realize we need to remove this in another task. Btw, I followed the Figma guide which defined the height.
![Screen Shot 2564-06-21 at 14 01 22](https://user-images.githubusercontent.com/29707647/122720044-2bb45580-d299-11eb-95de-df878e4711d7.png)